config/genesis/genesis_test.go

```diff
@@ -9,22 +10,30 @@ import (
 )
 
 func TestParseGenesisJson(t *testing.T) {
+	// Create temp file
+	file, err := ioutil.TempFile("", "genesis-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(file.Name())
+
 	expected := &Genesis{
 		Name:       "gossamer",
 		Id:         "gossamer",
 		Bootnodes:  []string{"/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj"},
 		ProtocolId: "gossamer",
```

Comment:
ID and ProtocolID here as well.

Reply:
this is already addressed in #555 and this code was not modified